python/ray/_private/gcs_pubsub.py

```diff
@@ -23,6 +25,8 @@
 
 logger = logging.getLogger(__name__)
 
+MAX_GCS_PUBLISH_RETRIES = 60
```

Comment:
How about adding environment config?  60 is default

Reply:
since the GCS server timeout retries are in many different places, I think we may need to think of a general solution to all of them and put them in one patch.